package/gluon-config-mode-contact-info/luasrc/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -1,21 +1,22 @@
 local cbi = require "luci.cbi"
 local i18n = require "luci.i18n"
 local uci = luci.model.uci.cursor()
+local site = require 'gluon.site_config'
 
 local M = {}
 
 function M.section(form)
   local s = form:section(cbi.SimpleSection, nil, i18n.translate(
-    'You can provide your contact information here to '
-      .. 'allow others to contact you. Please note that '
+    'Please provide your contact information here to '
+      .. 'allow others to contact you. Note that '
       .. 'this information will be visible <em>publicly</em> '
       .. 'on the internet together with your node\'s coordinates.'
     )
   )
 
   local o = s:option(cbi.Value, "_contact", i18n.translate("Contact info"))
   o.default = uci:get_first("gluon-node-info", "owner", "contact", "")
-  o.rmempty = true
+  o.rmempty = not site.owner.obligatory
```

Comment:
ok, since I refactored the option to `site.config_mode.owner.obligatory`  I changed this to 

      o.rmempty = not ((site.config_mode or {}).owner or {}).obligatory